components/Spinner.js

```diff
@@ -0,0 +1,13 @@
+import {React} from 'react';
+
+
+const Spinner =()=>{
+    return (
+    <div class="h-screen bg-white">
+    <div class="flex justify-center items-center h-full">
+      <img class="h-16 w-16" src="https://icons8.com/preloaders/preloaders/1488/Iphone-spinner-2.gif" alt="" />
+    </div>
+    </div>)
+}
+
+export default Spinner;
```

Comment:
Thanks for this component! Thinking to keep it in a `basic` folder named in the component folder!